wisp-task/README.md

```diff
@@ -0,0 +1,82 @@
+# wisp-task
+
+This module contains code to assist with running tasks.
+
+[RepeatedTask](https://github.com/cashapp/misk/blob/master/wisp-task/src/main/kotlin/wisp/task/RepeatedTask.kt)
+runs a repeated task at the user controlled rate based on the
+[kotlin-retry library](https://github.com/michaelbull/kotlin-retry).  For convenience a
+[RepeatedTaskManager](https://github.com/cashapp/misk/blob/master/wisp-task/src/main/kotlin/wisp/task/RepeatedTaskManager.kt)
+is available to help manage the
+[RepeatedTasks](https://github.com/cashapp/misk/blob/master/wisp-task/src/main/kotlin/wisp/task/RepeatedTask.kt).
+
+It's possible to specify your own retry policy for the task.  See
+[kotlin-retry library](https://github.com/michaelbull/kotlin-retry) for more details and examples.
+The default retry policy instructions are to retry on all exceptions with a Binary Exponential
+backoff delay - which itself is configured from the supplied
+[RepeatedTaskConfig](https://github.com/cashapp/misk/blob/master/wisp-task/src/main/kotlin/wisp/task/RepeatedTaskConfig.kt).
+
+## Usage
+
+Create a repeated task and start it running.
+
+```kotlin
+val taskConfig = TaskConfig("taskName")
+val repeatedTaskConfig = RepeatedTaskConfig(
+  timeBetweenRunsMs = 10000L  // 10 sec delay between task runs or retries
+)
+
+val manager = RepeatedTaskManager()
+
+val newTask = manager.createTask(
+  name = taskConfig.name,
```

Comment:
why take in a name if taskConfig also has a name, and taskConfig is required?

Reply:
put a different way, is taskConfig really required if all it's using is the name? is there a test or sample code to show?